.circleci/config.yml

```diff
@@ -312,6 +312,10 @@ jobs:
       - run: yarn gulp ci:build-release-packages
       - run: yarn check-release-output
 
+      # TODO(devversion): replace this with bazel tests that run Madge. This is
+      # cumbersome and doesn't guarantee no circular deps for other entry-points.
+      - run: yarn madge --circular dist/releases/cdk/schematics/index.js
```

Comment:
I decided to just add this one-liner as that command helped me finding the circular dependency that broke the Material update. Long-term solution should definitely be different.